advanced/Scripts/piholeCheckout.sh

```diff
@@ -166,12 +166,15 @@ checkout() {
         checkout_pull_branch "${webInterfaceDir}" "${2}"
     elif [[ "${1}" == "ftl" ]] ; then
         local path
+        local oldbranch
         path="${2}/${binary}"
+        oldbranch=$(pihole-FTL -b)
```

Comment:
```suggestion
        oldbranch="$(pihole-FTL -b)"
```

Branches are allowed to have spaces in them.